api/src/main/java/org/openmrs/module/muzima/handler/DemographicsUpdateQueueDataHandler.java

```diff
@@ -48,6 +48,11 @@
     private QueueProcessorException queueProcessorException;
     private Patient unsavedPatient;
 
+    /**
+     * 
+     * @param queueData - QueueData
```

Comment:
Needs description of what the method is doing and more description of the quedata param.